public/api-docs/swaggerDoc.json

```diff
@@ -447,9 +525,37 @@
                 "password": {
                     "type": "string"
                 },
-                "address": {
+                "residencAddress": {
+                    "type": "string",
+                    "description": "residence address of the user"
+                },
+                "department": {
+                    "type": "string",
+                    "description": "User department"
+                },
+                "lineManager": {
+                    "type": "string",
+                    "description": "line manager to the user"
+                },
+                "gender": {
+                    "type": "string",
+                    "description": "User gender"
+                },
+                "phoneNumber": {
+                    "type": "integer",
+                    "description": "Phone number of user"
+                },
+                "role": {
+                    "type": "string",
```

Comment:
This is not a string, this should be an integer

Reply:
Noted

Reply:
Done